airflow/kubernetes/kube_config.py

```diff
@@ -57,6 +57,14 @@ def __init__(self):
         # create, watch, get, and delete pods in this namespace.
         self.kube_namespace = conf.get(self.kubernetes_section, "namespace")
         self.multi_namespace_mode = conf.getboolean(self.kubernetes_section, "multi_namespace_mode")
+        if self.multi_namespace_mode and conf.get(
+            self.kubernetes_section, "multi_namespace_mode_namespace_list"
+        ):
+            self.multi_namespace_mode_namespace_list = conf.get(
+                self.kubernetes_section, "multi_namespace_mode_namespace_list"
+            ).split(",")
+        else:
+            self.multi_namespace_mode_namespace_list = None
```

Comment:
```suggestion
        self.multi_namespace_mode_namespace_list = conf.getjson(
            self.kubernetes_section, "multi_namespace_mode_namespace_list"
        )
```
maybe this would be a good use of json. or could add a `getlist`

Reply:
Hi @dstandish , I'm not sure if this is the correct thing to do. 

I think it's easier to allow users pass a value like `ns-a,ns-b,ns-c` to `multi_namespace_mode_namespace_list`, rather than asking them to put something like `"ns-a","ns-b","ns-c"`. So treating it as a json or list may not be making sense IMHO, and I would suggest we keep the existing way.

Let me know how you find it?

Reply:
sure